operating/manage-user-access/authentication-provider-structure.adoc

```diff
@@ -0,0 +1,93 @@
+:_content-type: ASSEMBLY
+[id="authentication-provider-structure"]
+= Authentication provider structure
+include::modules/common-attributes.adoc[]
+:context: authentication-provider-structure
+
+toc::[]
+
+[role="_abstract"]
+
+An authentication provider connects to a third-party source of user identity, for example, an identity provider or IDP, gets the user identity, issues a token based on that identity, and returns the token to RHACS. 
+This token allows RHACS to authorize the user. 
+RHACS uses the token within the user interface and API calls.
+
+After installing RHACS, you must set up your IDP to authorize users. 
+
+Navigate to *Operating -> Managing user access*, and select the appropriate module to set up your own IDP immediately after provisioning.
+
```

Comment:
```suggestion

[IMPORTANT]
====
If you are using OpenID Connect (OIDC) as an idP, {product-title-short} relies on mapping rules that examine the values of specific claims like `groups`, `email`, `userid` and `name` from either the user ID token or the `UserInfo` endpoint response to authorize the users.
====

If these details are absent, the mapping cannot succeed and the user does not get access to the required resources.
Therefore, you need to ensure that the claims required to authorize users from your IDP, for example, `groups`, are included in the authentication response of your IDP to enable successful mapping.

.Example

To connect an Azure Active Directory (AD) to {product-title-short} using  Sign-On (SSO) configuration, you need to add specific claims, for example, `groups` claim to tokens, and assign users, groups, or both to the enterprise application.
For instructions, see link:https://learn.microsoft.com/en-us/azure/active-directory/hybrid/how-to-connect-fed-group-claims#add-group-claims-to-tokens-for-saml-applications-using-sso-configuration[Add group claims to tokens for SAML applications using SSO configuration].
```

Reply:
To be honest I think the Azure example, even though it is a short one, should go on a separate page under `Configuring auth providers`. This would allow for better discoverability compared to

Reply:
ok, SGTM!

Please add the below content to your PR: 

> :_content-type: ASSEMBLY
> [id="connecting-azure-ad-rhacs-using-sso-configuration"]
> = Connecting Azure AD to {product-title-short} using SSO configuration
> include::modules/common-attributes.adoc[]
> :context: user-authentication
> 
> 
> [role="_abstract"]
> 
> To connect an Azure Active Directory (AD) to {product-title-short} using  Sign-On (SSO) configuration. you need to add specific claims, for example, `groups` claim to tokens, and assign users, groups, or both to the enterprise application.
> 
> 
> [id="adding-group-claims-to-tokens-for-saml-applications-using-SSO-configuration_{context}"]
> == Adding group claims to tokens for SAML applications using SSO configuration
> 
> Configure the application registration in Azure AD to include group claims in tokens. For instructions, see link:https://learn.microsoft.com/en-us/azure/active-directory/hybrid/how-to-connect-fed-group-claims#add-group-claims-to-tokens-for-saml-applications-using-sso-configuration[Add group claims to tokens for SAML applications using SSO configuration].

NOTE: I added a suggestion to update the topic.yaml file.